Source/WebCore/page/scrolling/mac/ScrollerPairMac.mm

```diff
@@ -81,15 +81,15 @@ - (BOOL)inLiveResizeForScrollerImpPair:(NSScrollerImpPair *)scrollerImpPair
 - (NSPoint)mouseLocationInContentAreaForScrollerImpPair:(NSScrollerImpPair *)scrollerImpPair
 {
     UNUSED_PARAM(scrollerImpPair);
-    if (!_scrollerPair)
-        return NSZeroPoint;
-
-    return _scrollerPair->lastKnownMousePosition();
+    // This location is only used when calling mouseLocationInScrollerForScrollerImp,
+    // where we will use the converted mouse position from the Web Process
```

Comment:
This comment also is only relevant to UI-side compositing.

Reply:
See above.